### PR TITLE
Updates for guide

### PR DESCRIPTION
**Pull request by taliaga**
Updating core recipes (mongo, orion, ql and iota) with checks from the guided-tour.

Removed bash-only script, preferring env var settings and plain docker stack deploy. This change is in the line of closer integration with generated files for portainer templates (and mastermind).

### CODE REVIEW

File: recipes/utils/mongo-replicaset/readme.md [hunk 1]
```diff
@@ -113,33 +114,33 @@ Let's add a new node to the swarm to see how docker deploys a new task of the
...
-    $ docker logs $(docker ps -f "name=mongo_mongo-controller" -q)
-    ...
-    INFO:__main__:To add: {'10.0.0.8'}
-    INFO:__main__:New config: {'version': 2, '_id': 'rs', 'members': [{'_id': 0, 'host': '10.0.0.5:27017'}, {'_id': 1, 'host': '10.0.0.3:27017'}, {'_id': 2, 'host': '10.0.0.4:27017'}, {'_id': 3, 'host': '10.0.0.8:27017'}]}
-    INFO:__main__:replSetReconfig: {'ok': 1.0}
+$ docker logs $(docker ps -f "name=mongo_mongo-controller" -q)
+...
+INFO:__main__:To add: {'10.0.0.8'}
+INFO:__main__:New config: {'version': 2, '_id': 'rs', 'members': [{'_id': 0, 'host': '10.0.0.5:27017'}, {'_id': 1, 'host': '10.0.0.3:27017'}, {'_id': 2, 'host': '10.0.0.4:27017'}, {'_id': 3, 'host': '10.0.0.8:27017'}]}
+INFO:__main__:replSetReconfig: {'ok': 1.0}
```
Comment (chicco785) [inline recipes/utils/mongo-replicaset/readme.md:143]: check rows 157 and 159 links are not working

File: recipes/tools/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (chicco785) [inline recipes/tools/build.sh:1]: so this is the way we keep then synch?

File: recipes/data-management/quantumleap/settings.bat [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@ECHO OFF
```
Comment (chicco785) [inline recipes/data-management/quantumleap/settings.bat:1]: is this documented in the readme?

File: recipes/data-management/quantumleap/settings.env [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Env variables to customize the recipe.
```
Comment (chicco785) [inline recipes/data-management/quantumleap/settings.env:1]: is this documented in the readme?

**Review by chicco785 — CHANGES_REQUESTED** [review]
what's the plan for the other services w.r.t. settings.env ? another PR?

File: recipes/tools/build.sh (hunk 2 above)
Comment (taliaga) [reply recipes/tools/build.sh:1]: Right. Developers take care of this, and clients of recipes only care about a single clone that brings everything. It's a copy, but it's automated. The `deploy_*.sh` scripts were making it difficult to integrate with dancn' approach for portainer templating.

File: recipes/data-management/quantumleap/settings.bat (hunk 3 above)
Comment (taliaga) [reply recipes/data-management/quantumleap/settings.bat:1]: Good, I forgot to mention these. We'd need a windows user to test these. Someone from @smartsdkInfotec maybe?

**Review by chicco785 — APPROVED** [review]

**Comment by chicco785** [comment]
there is still a linting issue https://travis-ci.org/smartsdk/smartsdk-recipes/jobs/372959940#L437

**Comment by taliaga** [comment]
@chicco785  answering your previous question that I almost missed, yes, I first needed to know daniele's updates to accomodate for portainer's templates. Adjustments will follow in new PR

**Pull request merged**